css/style.css

```diff
@@ -24,6 +24,27 @@ a.btn:hover
     background-color: lightgray;
 }
 
+/* replaces <big> tag */
+em.big
+{
+    font-style: normal;
+    font-size: larger;
+}
+
+/* replaces <tt> tag */
```

Comment:
I don't think we use the `code` tag currently. It does appear a few times in `contracts.dd` and once in `deprecate.dd` but I think that is not intentional.